<launch>
	<param name="/amcl/initial_pose_x"  value="2.40"/>
	<param name="/amcl/initial_pose_y"  value="1.30"/>
	<param name="/amcl/initial_pose_a"  value="-10.00"/>
</launch>
